tox.ini

```diff
@@ -1,24 +1,20 @@
 [tox]
 min_version = 4.0
-env_list =
-    py310
-    #type
-    #flake8
+env_list = testenv
 
 [gh-actions]
-python =
-    3.10: py310, type, flake8
+python = 3.10: py310, autopep8, type, flake8
 
 [testenv]
-deps = 
+deps =
     pytest
     pytest-cov
-commands = pytest tests
+    mypy
+    flake8
+    autopep8
 
-[testenv:type]
-deps = mypy
-commands = python -m mypy art #Withouth python -m it cries about third-party libraries
-
-[testenv:flake8]
-deps = flake8
-commands = flake8 art
\ No newline at end of file
+commands =
+    pytest tests
+    - autopep8 --in-place --recursive art
```

Comment:
I wonder if this makes sense to use autopep8 and then flake8. I assume that autopep8 doesn't create PR automatically. This test will always pass then but won't have influence on the repository. I'd rather prefer obtaining exception from flake8 and someone to manually run autopep8.

to be discussed, but this is why I requested changes

Reply:
agree